### DISCUSSION TITLE
What makes an admin?

### DISCUSSION THREAD

**Discussion by lordscarlet** [nathanwoulfe/Plumber-2#143] [Q&A] [answered]
We are trying to figure out why a particular user is being allowed to approve things "as admin". They are not in the administrator role in Umbraco. Our current hypothesis is that it is because the user has access to the Workflow section.

**Comment by nathanwoulfe** [comment, marked as answer]
Current hypothesis is correct - access to the workflow section makes a user a workflow admin.

**Reply by lordscarlet** [reply]
Hm, ok. So someone can't see the workflow status screens without being an admin. That is at best troublesome in our situation, if not a problem outright. I will have to think on it, but I may have to put in a request that someone can see the overall workflow requests screens without being an "admin" that can approve any and everything.

**Reply by nathanwoulfe** [reply]
Any requests need to go on the Umbraco Workflow issue tracker - Plumber is effectively no more. 

This would be a reasonably large change, as it's a core concept that access to the workflow section defines an admin user, so changing this logic would have substantial flow-on impacts.

I guess too I need to understand why a non-admin user needs to see all workflows?

**Reply by lordscarlet** [reply]
@nathanwoulfe I was completely unaware of the change to Umbraco Workflow! We need to get ourselves to v9/10, but keep having other priorities.

My concerns were alleviated when I was reminded of the "Workflow" dashboard. I use admin so much I was under the impression that was the only place to see that information. Because it is there, I rescind my thought that a change is needed.
